### ISSUE TITLE
Set up Windows Jenkins slave

### ISSUE THREAD

**Issue by jwillenbring** [trilinos/Trilinos#1158]
@trilinos/framework 

The Windows VM for Windows testing of Trilinos needs to be set up as a Jenkins slave so that we can automatically run jobs on it. Eventually we will have 2 such VMs, and only 1 should be used with Jenkins.

Done for this story is to have a functional Jenkins slave set up on one of the Trilinos Windows VMs.

**Comment by bmpersc** [comment]
I have gained access to the machine and am in the process of setting up software for it. I am running into some issues getting the necessary software, but am looking into ways of addressing that. I have been able to get some of the software, but not the most important, the compiler and MPI.

**Comment by bmpersc** [comment]
I have been able to install a lot of the software that is needed. However, I am currently running into issues with verifying that everything is working. As I went I did check that a sanity check worked, but that was fairly terse like <command> --version to make sure that it wasn't horribly broken. Currently all the required software besides clapack is installed in a SEMS like path on the machine. Clapack was the first piece of software that needed to be compiled. While trying to configure with cmake I run into an issue of the compiler not being found despite having run the vcvsvars32.bat script in the terminal in use.

Digging into this I found that the environment is not being setup properly. The path variable does not have the location of the VS compilers on it which is certainly part of the problem. I thought that using this non-standard install path might be at the heart of the issue of the script not properly setting up the environment but I don't think so at this point. I have put in some debugging statements in the vcvsvars32.bat script to see if it was just  assuming the install was in the "Program Files" directory and therefore failing for that reason. However, it appears that the script is written well enough to properly recognize that it was installed elsewhere and use that installation path for all of its work. From what I've seen so far everything it is doing is being done right and the setup is correct inside the script. However, once the script exits the terminal does not retain the settings. This could be related to the fact that we are using a windows server OS instead of the standard user version. The behavior I am seeing is more akin to what linux does for its scripts run in a subshell which is much safer behavior. I have not had an opportunity to look into if there is an equivalent to source for windows, but that could be a next step.
